.vscode/cspell.json

```diff
@@ -96,6 +96,15 @@
     {
       "filename": "**/eng/pipelines/templates/jobs/ci.yml",
       "words": ["warnaserror"]
+    },
+	{
+      "filename": "**/sdk/alertsmanagement/**/*.cs",
+      "words": [
+        "groupby",
+        "groupedby",
```

Comment:
since the spell check only checks the api file, we should rename these two properties so that we do not have to escape them here

Reply:
Unfortunately these 2 words are in the serialization code which are defined in the swagger, so we can rename from c# layer but can't change the REST payload